src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksVendorDataProcessingIntoKfsServiceImpl.java

```diff
@@ -85,22 +96,58 @@ private MaintenanceDocument createKfsVendorMaintenaceDocument(PaymentWorksVendor
         return vendorMaintenceDoc;
     }
 
-    private boolean kfsVendorMaintenanceDocumentValidated(MaintenanceDocument vendorMaintenceDoc, PaymentWorksNewVendorRequestsBatchReportData reportData, PaymentWorksVendor pmwVendor) {
+    private boolean kfsVendorMaintenanceDocumentValidatedForRoute(MaintenanceDocument vendorMaintenceDoc, PaymentWorksNewVendorRequestsBatchReportData reportData, PaymentWorksVendor pmwVendor) {
         boolean documentValidated = false;
         try {
             vendorMaintenceDoc.validateBusinessRules(new RouteDocumentEvent(vendorMaintenceDoc));
-            LOG.info("kfsVendorMaintenanceDocumentValidated: vendorMaintenceDoc validate.");
+            LOG.info("kfsVendorMaintenanceDocumentValidatedForRoute: vendorMaintenceDoc validate.");
+            documentValidated = true;
+        } catch (ValidationException ve) {
+            List<String> validationErrors = getPaymentWorksBatchUtilityService().convertReportDataValidationErrors(GlobalVariables.getMessageMap().getErrorMessages());
+            captureKfsProcessingErrorsForVendor(pmwVendor, reportData, validationErrors);
+            LOG.error("kfsVendorMaintenanceDocumentValidatedForRoute: eDoc validation error(s): " + validationErrors.toString());
+            LOG.error("kfsVendorMaintenanceDocumentValidatedForRoute: eDoc validation exception caught: " + ve.getMessage());       
+        } finally {
+            GlobalVariables.getMessageMap().clearErrorMessages();
+        }
+        return documentValidated;
+    }
+    
+    private boolean kfsVendorMaintenanceDocumentValidatedForSave(MaintenanceDocument vendorMaintenceDoc, PaymentWorksNewVendorRequestsBatchReportData reportData, PaymentWorksVendor pmwVendor) {
```

Comment:
There's a fair amount of duplicate logic between the save and route validation methods. If practical, it would be good to consolidate some of that setup.